pkg/che/che_test.go

```diff
@@ -525,6 +525,73 @@ func TestCheRequest(t *testing.T) {
 	})
 }
 
+func TestDeleteDevSpacesUser(t *testing.T) {
+	// given
+	config := commonconfig.NewMemberOperatorConfigWithReset(t, testconfig.Che().Namespace("crw").RouteName("devspaces"))
+	dbCleanerURL := "http://che-db-cleaner.crw/"
+
+	t.Run("success", func(t *testing.T) {
+		// given
+		cl, _ := prepareClientAndConfig(t, config)
+		cheClient := &Client{
+			httpClient: http.DefaultClient,
+			k8sClient:  cl,
+		}
+
+		// when
+		defer gock.OffAll()
+		gock.New(dbCleanerURL).
+			Delete("johnsmith456").
+			Persist().
+			Reply(200)
+		err := cheClient.DevSpacesDBCleanerDelete("johnsmith456")
```

Comment:
this is ok for tests, but the expected input is valid UUID (input like that would not pass the validation on the db cleaner end)

Reply:
ack, since we get the UID directly from the User resource in the code it should be fine